src/main/java/com/cburch/logisim/gui/log/OptionsPanel.java

```diff
@@ -299,11 +281,15 @@ public void componentShown(ComponentEvent e) {
       }
     });
 
+    // Close the frame once ESC is hit
+    frame.getRootPane().registerKeyboardAction(al -> frame.dispose(),
+            KeyStroke.getKeyStroke(KeyEvent.VK_ESCAPE, 0),
+            JComponent.WHEN_IN_FOCUSED_WINDOW);
   }
 
   static class ScrollablePanel extends JPanel implements Scrollable {
     /**
-     * 
+     *
```

Comment:
If the comment is empty, it can be removed entirely IMHO.

Reply:
Done

Reply:
Thanks!